src/front-end/src/components/MenteeFormAndEmbedder/MenteeFormAndEmbedder.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import MenteeForm from '../MenteeForm/MenteeForm';
+
+function MentorFormAndEmbedder() {
+  const [showEmbedCode, setShowEmbedCode] = useState(false);
+
+  function embed() {
+    setShowEmbedCode(!showEmbedCode);
+  }
+
+  return (
+    <div>
+      <div>
+        <MenteeForm />
+        <button onClick={embed} type="button">
+          Embed this form in your website
+        </button>
+      </div>
+      <div>
+        {showEmbedCode && (
+          <textarea
+            style={{
+              height: '150px',
+              resize: 'none',
+              marginTop: '10px',
+              backgroundColor: '#110a47',
+              color: 'white',
+            }}
+            value='<iframe width="1000" height="500" src="http://localhost:3000/mentee-form-embed" frameborder="0" allowfullscreen></iframe>'
```

Comment:
I don't think we'll have to worry about this since it's likely due to our specific linting setup, but: When I copied the code after clicking on the 'Embed this form in your website' button, I received the following error when I pasted the iFrame in a separate component. 
<img width="558" alt="Screen Shot 2021-02-01 at 6 54 34 PM" src="https://user-images.githubusercontent.com/20407302/106545832-eeeceb00-64be-11eb-8538-995aefbfda8a.png">
This was easily fixed by following what was displayed on the console to:
`<iframe
          title="test"
          width="1000"
          height="500"
          src="http://localhost:3000/mentee-form-embed"
          frameBorder="0"
          allowFullScreen
        />`
Since we're assuming that Admins will have some tech background (confirmed by the client) to be able to paste the iFrame tag into their organization's website's source code, I would also assume that they'd  be able to fix any linting errors that arise specific to their framework. So, I don't think this is an issue that we need to worry about, but I thought I would mention it incase it's worth discussing.

Reply:
This is interesting indeed. But this issue is quite specific to us (or anyone who would be using the same linting rules as us). This is valid html, it's just that we use react and eslint, so it's more finicky.